client/src/App.jsx

```diff
@@ -95,9 +97,11 @@ function App() {
           setMemorials={setMemorials}
           setLocation={setLocation}
         />
-        <ExpandButton type="button" onClick={() => setIsMainPage(!isMainPage)}>
-          <ExpandMoreIcon />
-        </ExpandButton>
+        {isMainPage && (
+          <ExpandButton type="button" onClick={() => setIsMainPage(!isMainPage)}>
+            <ExpandIcon src="/vector.svg" />
```

Comment:
Instead of placing assets in public folder, consider placing in directly in the project such as `~src/assets/images`, then import the image directly in the JS file.
See: https://create-react-app.dev/docs/adding-images-fonts-and-files/